src/core/Exception.pm

```diff
@@ -2462,9 +2462,10 @@ my class X::Multi::NoMatch is Exception {
                         ~  join("\n    ", '', sort keys @cand ∖ @un-rw-cand)
                     unless @cand == @un-rw-cand
                 )
-            !! join "\n    ",
-                'none of these signatures match:',
-                @cand
+            !! ( @cand
+                 ??  join "\n    ", 'none of these signatures match:', @cand
+                 !! "No signature match."
```

Comment:
The `No signature match` isn't very clear. Perhaps, it should read `Routine does not have any candidates. Is only the proto defined?`